solutions/USACO jan19-grass-planting.mdx

```diff
@@ -0,0 +1,75 @@
+---
+id: usaco-894
+source: USACO Silver 2019 January
+title: Grass Planting
+author: Neo Wang
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_planting_silver_jan19.html)
+
+**Time Complexity: $\mathcal{O}(N)$**
+
+<LanguageSection>
+
+<JavaSection>
+
+## Solution 1 (Java)
```

Comment:
I think time complexity should be directly beneath the solution header.

In other words, either move "Solution 1" up or move TC down. Also the thing in $$'s (I think) shouldn't be bolded. Just write `**Time Complexity:** $\mathcal{O}(N)$`